_data/merchants.yml

```diff
@@ -45,6 +45,8 @@
       url: https://shapeshift.io/
     - name: Tux Exchange
       url: https://www.tuxexchange.com/trade?coin=XMR&market=BTC
+      name: Trovemat.com (ATM)
```

Comment:
A `-` is missing in front of the name. Also I'm not sure if Trovemat should be considered an exchange by itself as they are a merchant selling ATMs.